src/Symfony/Component/Security/Http/AccessToken/Oidc/OidcTokenHandler.php

```diff
@@ -85,19 +84,16 @@ public function getUserBadgeFrom(string $accessToken): UserBadge
             $headerCheckerManager->check($jws, 0);
 
             // Verify the claims
-            $checkers = [
+            (new ClaimCheckerManager([
                 new Checker\IssuedAtChecker(clock: $this->clock, allowedTimeDrift: 0, protectedHeaderOnly: false),
                 new Checker\NotBeforeChecker(clock: $this->clock, allowedTimeDrift: 0, protectedHeaderOnly: false),
                 new Checker\ExpirationTimeChecker(clock: $this->clock, allowedTimeDrift: 0, protectedHeaderOnly: false),
                 new Checker\AudienceChecker($this->audience),
                 new Checker\IssuerChecker($this->issuers),
-            ];
-            $claimCheckerManager = new ClaimCheckerManager($checkers);
-            // if this check fails, an InvalidClaimException is thrown
-            $claimCheckerManager->check($claims);
+            ]))->check($claims);
 
             if (empty($claims[$this->claim])) {
```

Comment:
empty is dangerous and we try to avoid using it

Reply:
Should be fine in the last revision of this PR.
Thanks.